corehq/apps/accounting/models.py

```diff
@@ -1281,7 +1281,7 @@ def new_domain_subscription(cls, account, domain, plan_version,
         if date_end is not None:
             future_subscriptions = future_subscriptions.filter(date_start__lt=date_end)
         if future_subscriptions.count() > 0:
-            raise NewSubscriptionError(_(
+            raise NewSubscriptionError(unicode(_(
                 "There is already a subscription '%(sub)s' that has an end date "
```

Comment:
I think this is to force the execution of the lazy object rather than to force it into unicode